### PR TITLE
[CloudPC] Documentation update for cloudPcStatus.movingRegion

### PR DESCRIPTION
**Pull request by SleepIsImportant**

### CODE REVIEW

**Review by AshleyYangSZ — CHANGES_REQUESTED** [review]
Please add change log

**Comment by AshleyYangSZ** [comment]
The build failed for this PR, please help resolve.

**Review by AshleyYangSZ — APPROVED** [review]

**Review by RuiHou105 — APPROVED** [review]

File: api-reference/beta/resources/cloudpc.md [hunk 1]
```diff
@@ -98,6 +98,7 @@ Represents a cloud-managed virtual desktop. This Cloud PC is also enrolled into
 |pendingProvision|The provisioning is pending on the Cloud PC. In this case, the number of Cloud PCs in grace period is more than the number of total available licenses. |
 |restoring|The Cloud PC is restoring.|
 |unknownFutureValue|Evolvable enumeration sentinel value. Do not use.|
+|movingRegion|Indicates that Cloud PC is being moved from one region to another one.|
```
Comment (Danielabom) [inline api-reference/beta/resources/cloudpc.md:101]:
```suggestion
|movingRegion|Indicates that Cloud PC is being moved from one region to another.|
```

File: api-reference/beta/resources/cloudpc.md [hunk 2]
```diff
@@ -98,6 +98,7 @@ Represents a cloud-managed virtual desktop. This Cloud PC is also enrolled into
 |pendingProvision|The provisioning is pending on the Cloud PC. In this case, the number of Cloud PCs in grace period is more than the number of total available licenses. |
 |restoring|The Cloud PC is restoring.|
 |unknownFutureValue|Evolvable enumeration sentinel value. Do not use.|
+|movingRegion|Indicates that Cloud PC is being moved from one region to another.|
```
Comment (Danielabom) [inline api-reference/beta/resources/cloudpc.md:101]: In line 71 of the properties table, please add the "unknown future value" member and follow step 3 of this wiki: [Evolvable enums](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211749/Evolvable-enumerations?anchor=step-3%3A-add-clarification-if-one-or-more-members-follow-unknownfuturevalue)

**Review by Danielabom — CHANGES_REQUESTED** [review]
Hi @SleepIsImportant, please review my comments. thanks!

File: api-reference/beta/resources/cloudpc.md (hunk 2 above)
Comment (SleepIsImportant) [reply api-reference/beta/resources/cloudpc.md]: > In line 71 of the properties table (status), please add the "unknown future value" member and follow step 3 of this wiki: [Evolvable enums](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211749/Evolvable-enumerations?anchor=step-3%3A-add-clarification-if-one-or-more-members-follow-unknownfuturevalue)

fixed, thank you

File: api-reference/beta/resources/cloudpc.md [hunk 3]
```diff
@@ -73,7 +73,7 @@ Represents a cloud-managed virtual desktop. This Cloud PC is also enrolled into
 |servicePlanId|String|The service plan ID of the Cloud PC.|
 |servicePlanName|String|The service plan name of the Cloud PC.|
 |servicePlanType|[cloudPcServicePlanType](../resources/cloudpcserviceplan.md#cloudpcserviceplantype-values)|The service plan type of the Cloud PC.|
-|status|[cloudPcStatus](#cloudpcstatus-values)|The status of the Cloud PC. Possible values are: `notProvisioned`, `provisioning`, `provisioned`, `upgrading`, `inGracePeriod`, `deprovisioning`, `failed`, `restoring`.|
+|status|[cloudPcStatus](#cloudpcstatus-values)|The status of the Cloud PC. Possible values are: `notProvisioned`, `provisioning`, `provisioned`, `upgrading`, `inGracePeriod`, `deprovisioning`, `failed`, `restoring`,`movingRegion`, and `unknownFutureValue`.|
```
Comment (Danielabom) [inline api-reference/beta/resources/cloudpc.md:76]: @SleepIsImportant. The update is not correct. 
The value of the new member "movingRegion is greater than "unknown future value". And for that reason, you need to follow step 3 in this wiki [Evolvable-enumerations](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211749/Evolvable-enumerations)

File: api-reference/beta/resources/cloudpc.md (hunk 2 above)
Comment (Danielabom) [reply api-reference/beta/resources/cloudpc.md:105]: @SleepIsImportant, would you mind taking a second look at the wiki? The value movingRegion is greater than unknownFutureValue (based on what I see on the metadata), meaning there is a specific format for that. You will find the steps that you need to follow here: [Evolvable enums](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211749/Evolvable-enumerations?anchor=step-3%3A-add-clarification-if-one-or-more-members-follow-unknownfuturevalue)

File: api-reference/beta/resources/cloudpc.md [hunk 4]
```diff
@@ -73,7 +73,7 @@ Represents a cloud-managed virtual desktop. This Cloud PC is also enrolled into
 |servicePlanId|String|The service plan ID of the Cloud PC.|
 |servicePlanName|String|The service plan name of the Cloud PC.|
 |servicePlanType|[cloudPcServicePlanType](../resources/cloudpcserviceplan.md#cloudpcserviceplantype-values)|The service plan type of the Cloud PC.|
-|status|[cloudPcStatus](#cloudpcstatus-values)|The status of the Cloud PC. Possible values are: `notProvisioned`, `provisioning`, `provisioned`, `upgrading`, `inGracePeriod`, `deprovisioning`, `failed`, `restoring`.|
+|status|[cloudPcStatus](#cloudpcstatus-values)|The status of the Cloud PC. Possible values include: `notProvisioned`, `provisioning`, `provisioned`, `inGracePeriod`, `deprovisioning`, `failed`, `provisionedWithWarnings`, `resizing`, `pendingProvision`, `restoring`, `unknownFutureValue` and `movingRegion`.|
```
Comment (Danielabom) [inline api-reference/beta/resources/cloudpc.md:76]:
```suggestion
|status|[cloudPcStatus](#cloudpcstatus-values)|The status of the Cloud PC. The possible values are: `notProvisioned`, `provisioning`, `provisioned`, `inGracePeriod`, `deprovisioning`, `failed`, `provisionedWithWarnings`, `resizing`, `pendingProvision`, `restoring`, and `unknownFutureValue`. Note that you must use the `Prefer: include-unknown-enum-members` request header to get the following value in this `[evolvable enum](/graph/best-practices-concept#handling-future-members-in-evolvable-enumerations)`: `movingRegion`.|
```

**Comment by Danielabom** [comment]
PR approved.

**Pull request merged**